closed/OpenJ9.gmk

```diff
@@ -491,6 +496,12 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
     CMAKE_ARGS += -DCMAKE_C_COMPILER_LAUNCHER="$(ESCAPED_CCACHE)"
   endif # CCACHE
 
+  ifeq (true,$(OPENJ9_ENABLE_DDR))
+    CMAKE_ARGS += -DOMR_DDR=ON
+  else
+    CMAKE_ARGS += -DOMR_DDR=OFF
+  endif()
```

Comment:
`OPENJ9_ENABLE_DDR` is either `true` or `false` which I believe cmake understands, so this could be simplified to `-DOMR_DDR=$(OPENJ9_ENABLE_DDR)` in the initial definition of `CMAKE_ARGS`.

I'm not sure the parentheses are permitted on the `endif` line.